hakyll.cabal

```diff
@@ -56,10 +56,6 @@ Data-files:
   example/index.html
   example/about.rst
   example/contact.markdown
-  templates/atom-item.xml
-  templates/atom.xml
-  templates/rss-item.xml
-  templates/rss.xml
```

Comment:
These files need to be added to `Extra-source-files`, otherwise they don't can included in the tarball when running `cabal sdist`.